machines/scan.typegen.ts

```diff
@@ -51,7 +51,7 @@ export interface Typegen0 {
     actions: never;
     delays: never;
     guards: never;
-    services: 'QrLogin';
```

Comment:
Since scan.ts is removed, can we also remove the scan.typegen.ts ?

Reply:
removed